go.mod

```diff
@@ -1,7 +1,5 @@
 module github.com/spiffe/spire
 
-go 1.13
```

Comment:
Were these go.mod updates intentional? Not quite sure what impact they have...

Reply:
Not intentional, will remove in next commit.